src/oop/app/src/ScrapperFactory.php

```diff
@@ -7,13 +7,23 @@
 use src\oop\app\src\Parsers\FilmixParserStrategy;
 use src\oop\app\src\Transporters\CurlStrategy;
 use src\oop\app\src\Transporters\GuzzleAdapter;
+use Symfony\Component\DomCrawler\Crawler;
 use Exception;
 
 /** Class which decides which transporter and which parser use for site
  * ScrapperFactory
  */
 class ScrapperFactory
 {
+    private Movie $movie;
+
+    public function __construct()
+    {
+        $this->movieLeft = new Movie();
```

Comment:
why do you need to movies ? just pass new Movie in create method

Reply:
Done. It was a task not to change index.php